.github/workflows/build.yml

```diff
@@ -4,7 +4,6 @@ on:
   push:
```

Comment:
how about triggering a build on PR? I understand the rationale of building & pushing only on the main branch, but we need somehow to test the binary of the PR before merging. Is it handled in a different workflow?

Reply:
Fixed.